include/zephyr/bluetooth/sdp.h

```diff
@@ -29,175 +29,188 @@ extern "C" {
+#define BT_SDP_HCR_PRINT_SVCLASS            0x1126 /**< HCR Print */
+#define BT_SDP_HCR_SCAN_SVCLASS             0x1127 /**< HCR Scan */
+#define BT_SDP_CIP_SVCLASS                  0x1128 /**< Common ISDN Access */
+#define BT_SDP_VIDEO_CONF_GW_SVCLASS        0x1129 /**< Video Conferencing Gateway */
+#define BT_SDP_UDI_MT_SVCLASS               0x112a /**< UDI MT */
+#define BT_SDP_UDI_TA_SVCLASS               0x112b /**< UDI TA */
+#define BT_SDP_AV_SVCLASS                   0x112c /**< Audio/Video */
+#define BT_SDP_SAP_SVCLASS                  0x112d /**< SIM Access */
+#define BT_SDP_PBAP_PCE_SVCLASS             0x112e /**< Phonebook Access Client */
+#define BT_SDP_PBAP_PSE_SVCLASS             0x112f /**< Phonebook Access Server */
+#define BT_SDP_PBAP_SVCLASS                 0x1130 /**< Phonebook Access */
+#define BT_SDP_MAP_MSE_SVCLASS              0x1132 /**< Message Access Server */
+#define BT_SDP_MAP_MCE_SVCLASS              0x1133 /**< Message Notification Server */
+#define BT_SDP_MAP_SVCLASS                  0x1134 /**< Message Access Profile */
+#define BT_SDP_GNSS_SVCLASS                 0x1135 /**< GNSS */
+#define BT_SDP_GNSS_SERVER_SVCLASS          0x1136 /**< GNSS Server */
+#define BT_SDP_MPS_SC_SVCLASS               0x113a /**< MPS SC */
+#define BT_SDP_MPS_SVCLASS                  0x113b /**< MPS */
+#define BT_SDP_PNP_INFO_SVCLASS             0x1200 /**< PnP Information */
+#define BT_SDP_GENERIC_NETWORKING_SVCLASS   0x1201 /**< Generic Networking */
+#define BT_SDP_GENERIC_FILETRANS_SVCLASS    0x1202 /**< Generic File Transfer */
+#define BT_SDP_GENERIC_AUDIO_SVCLASS        0x1203 /**< Generic Audio */
+#define BT_SDP_GENERIC_TELEPHONY_SVCLASS    0x1204 /**< Generic Telephony */
+#define BT_SDP_UPNP_SVCLASS                 0x1205 /**< UPnP Service */
+#define BT_SDP_UPNP_IP_SVCLASS              0x1206 /**< UPnP IP Service */
+#define BT_SDP_UPNP_PAN_SVCLASS             0x1300 /**< UPnP IP PAN */
+#define BT_SDP_UPNP_LAP_SVCLASS             0x1301 /**< UPnP IP LAP */
+#define BT_SDP_UPNP_L2CAP_SVCLASS           0x1302 /**< UPnP IP L2CAP */
+#define BT_SDP_VIDEO_SOURCE_SVCLASS         0x1303 /**< Video Source */
+#define BT_SDP_VIDEO_SINK_SVCLASS           0x1304 /**< Video Sink */
+#define BT_SDP_VIDEO_DISTRIBUTION_SVCLASS   0x1305 /**< Video Distribution */
+#define BT_SDP_HDP_SVCLASS                  0x1400 /**< HDP */
+#define BT_SDP_HDP_SOURCE_SVCLASS           0x1401 /**< HDP Source */
+#define BT_SDP_HDP_SINK_SVCLASS             0x1402 /**< HDP Sink */
+#define BT_SDP_GENERIC_ACCESS_SVCLASS       0x1800 /**< Generic Access Profile */
+#define BT_SDP_GENERIC_ATTRIB_SVCLASS       0x1801 /**< Generic Attribute Profile */
+#define BT_SDP_APPLE_AGENT_SVCLASS          0x2112 /**< Apple Agent */
+/**
+ * @}
+ */
 
-/*
- * Attribute identifier codes
+/**
+ * @name Attribute identifier codes
+ * @{
  */
 #define BT_SDP_SERVER_RECORD_HANDLE 0x0000
```

Comment:
mmhh yeah, nice catch. I wonder if it could be that this constant shouldn't even be defined in the first place, and is a "dupe" of `BT_SDP_ATTR_RECORD_HANDLE`?
Just pushed a change to leave `BT_SDP_SERVER_RECORD_HANDLE` alone, and maybe other reviewers can confirm whether it's even needed in the first place.

Reply:
@jhedberg might know